tests/integ/test_marketplace.py

```diff
@@ -33,7 +34,6 @@
 from tests.integ.test_multidatamodel import (
     _ecr_image_uri,
     _ecr_login,
-    _create_repository,
```

Comment:
why? https://github.com/aws/sagemaker-python-sdk/blob/master/tests/integ/test_marketplace.py#L33-L38

Reply:
This was introduced in the beta Exp plus change.
The reason is at that time, similar to this `test_marketplace.py` file, our integ test needs to reuse the internal method `_create_repository` defined in another module (see below), I feel it's not appropriate so I moved this method to `from tests.integ.utils import create_repository` and make it as public method. Even if in M1 Exp plus, we simplified our integ tests and do not need to create repository, I still keep this change as I feel it's more appropriate to expose this useful helper method to utils as a public method.
```
from tests.integ.test_multidatamodel import (
    _ecr_image_uri,
    _ecr_login,
    _create_repository,
```